kustomize/components/google-cloud-operations/README.md

```diff
@@ -1,4 +1,4 @@
-# Integrating Online Boutique with Google Cloud Operations
+# Integrate Online Boutique with Google Cloud Operations
```

Comment:
**Rationale for change:**
I reworded a lot of headings such that they're imperative (e.g., "Do" instead of "Doing") — to be consistent with cloud.google.com documentation.